Python 3.11: tests/test_argparse_completer.py::test_add_parser_custom_completer fails

On Python 3.11.0b1, the following test fails:

```pytb
__________________________________________________ test_add_parser_custom_completer ___________________________________________________

    def test_add_parser_custom_completer():
        """Tests setting a custom completer type on a subcommand using add_parser()"""
        parser = Cmd2ArgumentParser()
        subparsers = parser.add_subparsers()
    
        no_custom_completer_parser = subparsers.add_parser(name="no_custom_completer")
        assert no_custom_completer_parser.get_ap_completer_type() is None  # type: ignore[attr-defined]
    
>       custom_completer_parser = subparsers.add_parser(name="no_custom_completer", ap_completer_type=CustomCompleter)

tests/test_argparse_completer.py:1374: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = _SubParsersAction(option_strings=[], dest='==SUPPRESS==', nargs='A...', const=None, default=None, type=None, choices={...rgparse_custom.Cmd2HelpFormatter'>, conflict_handler='error', add_help=True)}, required=False, help=None, metavar=None)
name = 'no_custom_completer'
kwargs = {'ap_completer_type': <class 'tests.test_argparse_completer.CustomCompleter'>, 'prog': 'pytest no_custom_completer'}
aliases = ()

    def add_parser(self, name, **kwargs):
        # set prog from the existing prefix
        if kwargs.get('prog') is None:
            kwargs['prog'] = '%s %s' % (self._prog_prefix, name)
    
        aliases = kwargs.pop('aliases', ())
    
        if name in self._name_parser_map:
>           raise ArgumentError(self, _('conflicting subparser: %s') % name)
E           argparse.ArgumentError: argument {no_custom_completer}: conflicting subparser: no_custom_completer

/usr/lib/python3.11/argparse.py:1185: ArgumentError
```

--- a/tests/test_argparse_completer.py
+++ b/tests/test_argparse_completer.py
@@ -1371,5 +1371,5 @@ def test_add_parser_custom_completer():
     no_custom_completer_parser = subparsers.add_parser(name="no_custom_completer")
     assert no_custom_completer_parser.get_ap_completer_type() is None  # type: ignore[attr-defined]
 
-    custom_completer_parser = subparsers.add_parser(name="no_custom_completer", ap_completer_type=CustomCompleter)
+    custom_completer_parser = subparsers.add_parser(name="custom_completer", ap_completer_type=CustomCompleter)
     assert custom_completer_parser.get_ap_completer_type() is CustomCompleter  # type: ignore[attr-defined]

